File: libs/common/models/vw-specialization-cost.ts



export class VwSpecializationCost {
    public specializationId: number = 0;
    public programId: number;
    public providerId: number;
    public sessionOfStudyId: number = 0;
    public fundingYearCode: string = '';
    public sessionStartDate: Date;
    public sessionEndDate: Date;
    public costType: string = '';
    public amount: number;
    public studyYear: number;
    public displayOnSfsflag: boolean;

    constructor(init? : Partial<VwSpecializationCost>) {
        
        Object.assign(this, init);
    }
}

File: apps/institution-api/Models/EntityFramework/VwSpecializationCost.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace ProviderApi.Models.EntityFramework
{
    public partial class VwSpecializationCost
    {
        public int SpecializationId { get; set; }
        public int? ProgramId { get; set; }
        public int? ProviderId { get; set; }
        public int SessionOfStudyId { get; set; }
        public string FundingYearCode { get; set; }
        public DateTime? SessionStartDate { get; set; }
        public DateTime? SessionEndDate { get; set; }
        public string CostType { get; set; }
        public decimal? Amount { get; set; }
        public decimal? StudyYear { get; set; }
        public bool? DisplayOnSfsflag { get; set; }
    }
}
